ftw/simplelayout/browser/resources/integration.theme.scss

```diff
@@ -427,6 +427,15 @@ ul[class^="sl-toolbar"] {
   }
 }
 
+.SimplelayoutPortlet {
+ .sl-column {
+  min-height: 100px;
+    &:empty {
+      border: 1px dashed $color-text;
```

Comment:
@maethu I guess the dashed border would be visible in anonymous mode too. This is why we attached the rule to `documentEditable`.